packages/react-native/src/primitives/Radio/types.ts

```diff
@@ -0,0 +1,38 @@
+import { StyleProp, TextStyle, ViewProps, ViewStyle } from 'react-native';
+
+import { LabelPosition } from '../Label/types';
+
+// This type should live elsewhere
+export type Sizes = 'small' | 'medium' | 'large';
```

Comment:
Union type naming should be singular form. Also let's add `number` to the union instead of declaring another union on ln 16

Reply:
Ok. Since we are likely to have a `size` prop for other primitives as well, I think we'll eventually want to find another place to define the `Sizes` type.